pkg/jobrunaggregator/jobrunhistoricaldataanalyzer/types.go

```diff
@@ -11,23 +11,30 @@ import (
 var prTemplate string
 
 type parsedJobData struct {
+	PercentTimeDiffP50                 float64       `json:"-"`
+	PercentTimeDiffP75                 float64       `json:"-"`
 	PercentTimeDiffP95                 float64       `json:"-"`
 	PercentTimeDiffP99                 float64       `json:"-"`
+	TimeDiffP50                        time.Duration `json:"-"`
+	TimeDiffP75                        time.Duration `json:"-"`
```

Comment:
Per above, we don't need to track time diffs for 50 or 75, we just need the data copied over.